### ISSUE TITLE
Specify path for bundle install

### ISSUE THREAD

**Issue by sasjo** [maierj/fastlane-action#19]
I'd like to cache installed gems to speed up my build. This would be easier if bundler is used with the `--path vendor/bundle` parameter. Without it, I'm not aware of a cross-platform location to cache.

This would also match the [Ruby cache example](https://github.com/actions/cache/blob/master/examples.md#ruby---gem).

```
      - name: Cache Ruby gems
        uses: actions/cache@v1
        with:
          path: vendor/bundle
          key: ${{ runner.os }}-gem-${{ hashFiles('**/Gemfile.lock') }}
          restore-keys: |
            ${{ runner.os }}-gem-
```

**Comment by maierj** [comment]
Would you suggest to hardcode the path or provide an optional parameter in the step configuration? I'd prefer to implement it as an optional parameter in order to not be too dependent on this cache example.

**Comment by sasjo** [comment]
An optional parameter to specify the path would be a good solution.

**Comment by maierj** [comment]
Implemented in Version 1.1.0.
